android/tangram/src/main/cpp/jniExports.cpp

```diff
@@ -367,7 +370,7 @@ extern "C" {
         assert(mapPtr > 0);
         auto map = reinterpret_cast<Tangram::Map*>(mapPtr);
         auto sourceName = stringFromJString(jniEnv, name);
-        auto source = std::shared_ptr<Tangram::TileSource>(new Tangram::ClientGeoJsonSource(sourceName, ""));
+        auto source = std::shared_ptr<Tangram::TileSource>(new Tangram::ClientGeoJsonSource(map->getPlatform(), sourceName, ""));
```

Comment:
Ok, I see the use of `getPlatform` here (In reference to my previous comment at the declaration of `getPlatform`).

Probably it make sense to abstract out these methods to apt `Map` functions.

Example: 
`map->createClientSource(sourceName, "");`